app/controllers/concerns/course_basic_detail_concern.rb

```diff
@@ -144,6 +145,8 @@ def course_creation_path_for(page)
       new_provider_recruitment_cycle_courses_study_mode_path(path_params)
     when :applications_open
       new_provider_recruitment_cycle_courses_applications_open_path(path_params)
+    when :accredited_body
+      new_provider_recruitment_cycle_courses_accredited_body_path(path_params)
```

Comment:
CodeClimate is complaining about this methods being too long. For this particular instance, I don't think we should change anything ... they're a little too long to turn into one-liners (`when ... then ...`).